evaluator/src/test/kotlin/ProjectSourceRuleTest.kt

```diff
@@ -142,3 +177,53 @@ private fun File.addDirs(vararg paths: String) {
         resolve(path).mkdirs()
     }
 }
+
+private fun ortResultWithDetectedLicenses(vararg detectedLicensesForFilePath: Pair<String, Set<String>>): OrtResult {
+    val id = Identifier("Maven:org.oss-review-toolkit:example:1.0")
+    val vcsInfo = VcsInfo(
+        type = VcsType.GIT,
+        url = "https://github.com/oss-review-toolkit/example.git",
+        revision = "0000000011111111000000001111111100000000"
```

Comment:
For easier search & replace operations throughout the code base, we aligned on using all zeros for dummy SHA1s.